app/components/handle-input/component.js

```diff
@@ -36,6 +36,9 @@ export default FocusableInput.extend({
     },
     set(key, value) {
       this._sanitizedValue = sanitizeInput(value);
+
+      this.attrs.value.update(this._sanitizedValue);
```

Comment:
Yes, this was the simplest possible work around to the ember bug we looked at yesterday (linked in description above). All of the other methods I attempted had other downsides. 

In the long run I plan to remove this work around, and move away from two way bindings. Instead the component will just call an action when it's value updates (instead of relying on two way bindings to do the mutation).

The bug is basically that setter CP's do not push their values upstream to mutate properly. This work around is manually calling that upstream update on the attrs object.